src/runtime.js

```diff
@@ -9,7 +9,7 @@ export default async function(config) {
   let db = await createConnection(config.mongo.connectionString);
   let jobs = kue.createQueue({
     prefix: config.kue.prefix,
-    redis: config.redis
+    redis: config.redis.connectionString
```

Comment:
Based on no knowledge of Joi at all, it looks like this used to be `redis: {host: "somehostname"}`, but now it will be `redis: "someConnectionString"`.